packages/api-server/api_server/default_config.py

```diff
@@ -18,5 +18,8 @@
     "oidc_url": None,
     # client id registered with the authentication provider, this will be used to verify the
```

Comment:
This comment probably should be updated, the aud is no longer same as the client_id.